common/scala/src/main/scala/org/apache/openwhisk/core/entity/Identity.scala

```diff
@@ -129,10 +131,12 @@ object Identity extends MultipleReadersSingleWriterCache[Option[Identity], DocIn
       stale = StaleParameter.No)
   }
 
-  private def rowToIdentity(row: JsObject, key: String)(implicit transid: TransactionId, logger: Logging) = {
+  protected[entity] def rowToIdentity(row: JsObject, key: String)(implicit transid: TransactionId, logger: Logging) = {
```

Comment:
added a unit test for this method (there was a bug in the one of the test helper methods).